plugins/nvm/_nvm

```diff
@@ -1,7 +1,9 @@
 #compdef nvm
 #autoload
 
-[[ -s ~/.nvm/nvm.sh ]] || return 0
+if ![ -f $HOME/.nvm/nvm.sh ] && ![ $commands[brew] -a -f `brew --prefix`/etc/profile.d/autojump.sh ]; then
```

Comment:
`autojump.sh`? Should this be testing for `nvm.sh` instead?

OMZ generally prefers the double-bracket `[[ ... ]]` test syntax to the single-bracket `[ ... ]` syntax.